butterknife/src/main/java/butterknife/internal/Utils.java

```diff
@@ -22,6 +22,8 @@ private static boolean hasSupportV4() {
       return true;
     } catch (ClassNotFoundException ignored) {
       return false;
+    } catch (VerifyError ignored) {
```

Comment:
I dunno, it happened to me locally though which is why I added it. I get the following if I remove it:

```
WARNING: no system properties value for ro.build.date.utc
DEBUG: Loading resources for android from jar:/Users/zsweers/.m2/repository/org/robolectric/android-res/4.1.2_r1_rc/android-res-4.1.2_r1_rc-real.jar!/res...

java.lang.VerifyError: Expecting a stackmap frame at branch target 46
Exception Details:
  Location:
    android/support/v4/graphics/drawable/DrawableCompat.<init>()V @18: ifnull
  Reason:
    Expected stackmap frame at this location.
  Bytecode:
    0x0000000: 2ab7 0093 2ab6 0096 1298 0312 02b8 009e
    0x0000010: 4c2b c600 1c2b 2a2a b600 a203 bd00 04b9
    0x0000020: 00a8 0400 57a7 0014 4d2c b800 acbf 2ab7
    0x0000030: 0090 b14d 2cb8 00ac bfb1               
  Exception Handler Table:
    bci [21, 37] => handler: 40
    bci [46, 50] => handler: 51


	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at butterknife.internal.Utils.hasSupportV4(Utils.java:21)
	at butterknife.internal.Utils.<clinit>(Utils.java:17)
	at com.example.butterknife.unbinder.A$$ViewBinder.bind(A$$ViewBinder.java:21)
	at com.example.butterknife.unbinder.B$$ViewBinder.bind(B$$ViewBinder.java:17)
	at com.example.butterknife.unbinder.C$$ViewBinder.bind(C$$ViewBinder.java:19)
	at com.example.butterknife.unbinder.E$$ViewBinder.bind(E$$ViewBinder.java:17)
	at com.example.butterknife.unbinder.G$$ViewBinder.bind(G$$ViewBinder.java:19)
	at com.example.butterknife.unbinder.H$$ViewBinder.bind(H$$ViewBinder.java:18)
	at com.example.butterknife.unbinder.H$$ViewBinder.bind(H$$ViewBinder.java:14)
	at butterknife.ButterKnife.bind(ButterKnife.java:198)
	at butterknife.ButterKnife.bind(ButterKnife.java:178)
	at com.example.butterknife.unbinder.A.<init>(A.java:14)
	at com.example.butterknife.unbinder.B.<init>(B.java:14)
	at com.example.butterknife.unbinder.C.<init>(C.java:16)
	at com.example.butterknife.unbinder.E.<init>(E.java:14)
	at com.example.butterknife.unbinder.G.<init>(G.java:17)
	at com.example.butterknife.unbinder.H.<init>(H.java:16)
	at com.example.butterknife.unbinder.UnbinderTest.verifyContentViewBinding(UnbinderTest.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.robolectric.RobolectricTestRunner$2.evaluate(RobolectricTestRunner.java:234)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.robolectric.RobolectricTestRunner$1.evaluate(RobolectricTestRunner.java:175)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:234)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:74)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:144)


Process finished with exit code 255
```